### ISSUE TITLE
Display the release date on each release notes page

### ISSUE THREAD

**Issue by roryokane** [ziglang/www.ziglang.org#149]
## Problem

Pages like https://ziglang.org/download/0.8.1/release-notes.html and https://ziglang.org/download/0.8.0/release-notes.html don’t display the date of those releases:

<img width="551" alt="release notes without date" src="https://user-images.githubusercontent.com/79168/132994699-4a7f10aa-02b2-4d11-8ad1-542e3384e8ff.png">

That is information I look for on a release notes page.

## Proposed solution

I think the best place for the release date would be this heading:

https://github.com/ziglang/www.ziglang.org/blob/9de50d6ace53ac93106ced15073daf2c8691a612/src/download/0.8.1/release-notes.html#L219

Making the release notes page look like this:

<img width="553" alt="release notes with date in heading next to version number" src="https://user-images.githubusercontent.com/79168/132994703-184ff588-fbbe-4963-9e28-70a17bd92b1f.png">

I haven’t found any template for all release notes files – I only found individual files for each release, such as [src/download/0.8.1/release-notes.html](https://github.com/ziglang/www.ziglang.org/blob/9de50d6ace53ac93106ced15073daf2c8691a612/src/download/0.8.1/release-notes.html) and [src/download/0.8.0/release-notes.html](https://github.com/ziglang/www.ziglang.org/blob/9de50d6ace53ac93106ced15073daf2c8691a612/src/download/0.8.0/release-notes.html). So the above heading might have to be edited in each release-notes.html file.

### Related code to reference when implementing

In this codebase, release dates are stored in [data/releases.json](https://github.com/ziglang/www.ziglang.org/blob/master/data/releases.json).

The code to build https://ziglang.org/download/ includes release dates on its page. It gets the date like this:

https://github.com/ziglang/www.ziglang.org/blob/9de50d6ace53ac93106ced15073daf2c8691a612/themes/ziglang-original/layouts/_default/downloads.html#L11-L12

https://github.com/ziglang/www.ziglang.org/blob/9de50d6ace53ac93106ced15073daf2c8691a612/themes/ziglang-original/layouts/_default/downloads.html#L16

**Comment by nektro** [comment]
That info is available here https://github.com/ziglang/zig/releases

It is bad practice to tie a release to a date.

**Comment by Atomk** [comment]
@nektro Any evidence supporting that claim? [Rust](https://releases.rs/), [Go](https://go.dev/doc/devel/release), [Python](https://docs.python.org/3/whatsnew/changelog.html), [Swift](https://www.swift.org/download/), even [release pages on Github](https://github.com/ziglang/zig/releases) all clearly show the release date.

It's useful information to contextualize the content of the page (i.e. some blog post linked to the 0.11.0 release notes and readers can easily see when that happened), it doesn't make sense to waste readers' time by having them look somewhere else for something this trivial.

For the same reason I am very glad that the News page (and posts finally) have dates too.

**Comment by nektro** [comment]
i disagree with my previous statement in this context. i might have misunderstood the request at first
